soc/arm/microchip_mec/mec172x/Kconfig.defconfig.mec172xnsz

```diff
@@ -39,6 +39,10 @@ config ESPI_XEC_V2
 	default y
 	depends on ESPI
 
+config ESPI_SAF_XEC_V2
```

Comment:
Similar  change is missing for mec152x
./soc/arm/microchip_mec/mec1501/Kconfig.defconfig.mec1501hsz

With updated KConfig.xec ESPI_SAF_XEC is never set when build MEC15xx boards

Reply:
fixed